src/platform/landing-pages/dev-template.ejs

```diff
@@ -121,16 +118,20 @@
       </div>
     </header>
 
-    <!-- TODO: Add breadcrumbs -->
-
     <script type="text/javascript">
     (function() {
       var module = {};
-      {% include "src/site/assets/js/incompatible-browser.js" %}
+      <%= inlineScripts['incompatible-browser.js'] %>
       checkBrowserCompatibility();
     })();
     </script>
 
+    <!-- TODO: Add breadcrumbs -->
+    <!--
+    <nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
+    </nav>
+    -->
```

Comment:
I thought the app manifests would have the `includeBreadcrumbs` attribute, but that's no longer the case.

It is available in `registry.json`, but I wasn't sure about depending on a file that's getting from the repo in the near future.

In any case, the lack of breadcrumbs is the reason for having no padding between the main content and the header (as seen in the screenshot).